tests/unit/beacons/test_status.py

```diff
@@ -45,14 +46,32 @@ def setup_loader_modules(self):
     def test_empty_config(self, *args, **kwargs):
         config = {}
         ret = status.beacon(config)
-        self.assertEqual(sorted(list(ret[0]['data'])), sorted(['loadavg', 'meminfo', 'cpustats', 'vmstats', 'time']))
+
+        if sys.platform.startswith('win'):
```

Comment:
Is there any particular reason you're using `sys` here instead of `salt.utils.is_windows()`?

Reply:
It's related to an issue I had fixing `unit.utils.test_verify` where importing `salt.utils` was creating a circular import, so I put it behind a try block or something... Mike recommended I use `sys.platform` there instead. It may not apply in this case, but I just started using `sys.platform`. Here's the PR I'm referencing: https://github.com/saltstack/salt/pull/43314#discussion_r138138101